app/jobs/sms_otp_sender_job.rb

```diff
@@ -0,0 +1,20 @@
+class SmsOtpSenderJob < ActiveJob::Base
+  queue_as :sms
+
+  def perform(code:, phone:, otp_created_at:)
+    otp_invalid?(otp_created_at) || send_otp(TwilioService.new, code, phone)
```

Comment:
how about
```ruby
otp_valid?(otp_created_at) && send_otp(TwilioService.new, code, phone)
```

otherwise it feels like asserting a negative (like `if !` or `unless`)